mypy/options.py

```diff
@@ -121,6 +121,7 @@ def __init__(self) -> None:
         self.auto_baseline = True
         self.default_return = False
         self.targets: List[str] = []
+        self.ignore_any_from_error = True
```

Comment:
why is it defaulted to `True` here but `False` here https://github.com/KotlinIsland/basedmypy/pull/281/files#diff-207f26497af1585033935dce5b1b9e3a12f9b98f28a8a8d7bf7bbb1ca88375aeR549

Reply:
`False` means `True` in the trippple negative cli framework.

Reply:
epic